### PR TITLE
[CORL-2945]: Update links in docs

### PR DESCRIPTION
**Pull request by kabeaty**
## What does this PR do?

These changes update the links in the docs so that they all go to the expected location post-monorepo updates.

## These changes will impact:

- [ ] commenters
- [ ] moderators
- [ ] admins
- [x] developers

## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags?

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Look over the docs, see that links are now correct.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ verify features introduced or bugs fixed in this PR.
 
  -->
 
-## Where any tests migrated to React Testing Library?
+## Were any tests migrated to React Testing Library?
```
Comment (nick-funk) [inline .github/PULL_REQUEST_TEMPLATE.md:73]: Oh my goodness, thank you! This typo hollers in my head every time I submit a PR 🤣

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -126,8 +126,8 @@ This can ensure that code reviews contain more meaningful feedback tied to the
 contribution rather than nit-picking on stylistic choices.
 
 Reviewers must ensure that linting and tests pass in CI and locally prior to a
-review taking place. You can do this by running `npm run generate` followed by
-`npm run lint` and `npm run test`.
+review taking place. You can do this by running `sh scripts/generate.sh` followed by
+`sh scripts/lint.sh`. You can run `npm run test` in `/server` and `/client`.
```
Comment (nick-funk) [inline CONTRIBUTING.md:130]: Thank you for fixing this up!

File: docs/docs/events.md [hunk 3]
```diff
@@ -13,11 +13,11 @@ In this example, we’re logging all events to the console as well as sending th
...
   const CoralStreamEmbed = Coral.createStreamEmbed({
-    events: function(events) {
-      events.onAny(function(eventName, data) {
+    events: function (events) {
+      events.onAny(function (eventName, data) {
         console.log(eventName, data);
-        if (eventName === 'createComment.success') {
-          my_event_tracker.send('createComment', data);
+        if (eventName === "createComment.success") {
+          my_event_tracker.send("createComment", data);
```
Comment (nick-funk) [inline docs/docs/events.md:20]: I too, prefer `"` over `'` 👍

File: docs/docs/migrate-data.md [hunk 4]
```diff
@@ -59,7 +59,7 @@ export CORAL_SITE_ID=""
 
 # This importer tool is designed to work with Coral at the following migration
 # version. This is the newest file in the
-# https://github.com/coralproject/talk/tree/develop/src/core/server/services/migrate/migrations
```
Comment (nick-funk) [inline docs/docs/migrate-data.md]: I wonder why this was pointing at `develop` previously... oh well, targeting `main` is probably the right call though!

**Review by nick-funk — APPROVED** [review]
Looks great, approved! ✅

**Review by marcushaddon — COMMENTED** [review]
I noticed when building the docs, I needed to run `sh scripts/build-development.sh` rather than `sh build-development`, which I think needs to be updated in `docs/README.md`.

**Review by nick-funk — APPROVED** [review]

**Pull request merged**